webhook/utils.ts

```diff
@@ -0,0 +1,184 @@
+import { access, copyFile, mkdir } from 'fs/promises'
+import { constants } from 'fs'
+import path from 'path'
+
+import { readJSON, remove, writeJSON } from 'fs-promise'
+import execa from 'execa'
+import { v4 as uuidv4 } from 'uuid'
+
+const { TEMP_DIR } = require('./consts.json')
+
+// Types
+export type WebhookConfig = {
+  gtfsFeeds: Array<{ uri: string; name: string; filename: string }>
+  csvFiles: Array<string>
+  workerId: string
+  deploymentId: string
+  logUploadUrl: string
```

Comment:
Do we have some linting rule about alphabetizing TypeScript props? If not we should probably add it.

Reply:
Have added a rule in https://github.com/ibi-group/tsdx/pull/1, and applied that rule in 5e19b289b0008824f323fc853327445a030e5e4f